install-nhc.sh

```diff
@@ -70,12 +70,12 @@ distro_check=$( cat /etc/os-release | grep -i ID_LIKE=)
 distro=`awk -F= '/^NAME/{print $2}' /etc/os-release`
 
 if [[ $distro_check =~ "debian" ]]; then
-  sudo DEBIAN_FRONTEND=noninteractive apt-get install -y libpci-dev hwloc build-essential libboost-program-options-dev libssl-dev plocate cmake
+  sudo DEBIAN_FRONTEND=noninteractive apt-get install -y libpci-dev hwloc build-essential libboost-program-options-dev libssl-dev cmake $(apt-cache search libpci-dev | grep -o '^[^ ]*')
 elif [[ $distro =~ "AlmaLinux" ]]; then
-  sudo dnf install -y pciutils-devel hwloc openssl-devel boost-devel mlocate cmake
+  sudo DEBIAN_FRONTEND=noninteractive apt-get install -y pciutils-dev hwloc libssl-dev libboost-dev cmake
 # this will attempt to install for the following os's: CentOS, RHEL, Fedora
 elif [[ $distro_check =~ "CentOS" ]] || [[ $distro_check =~ "Rhel" ]] || [[ $distro_check =~ "fedora" ]]; then
-  sudo yum install -y  pciutils-devel hwloc openssl-devel boost-devel mlocate cmake > /dev/null
+  sudo DEBIAN_FRONTEND=noninteractive apt-get install -y pciutils-dev hwloc libssl-dev libboost-dev cmake > /dev/null
```

Comment:
apt-get does not work on these distribution types. Please revert the change and use the previous line excluding the locate package. Yum is the package manager we want to use.